pynailgun/ng.py

```diff
@@ -898,26 +900,52 @@ def main():
         cmd = os.path.basename(sys.argv[0])
 
     if options.help_set and not args:
-        cmd_args = "help"
+        cmd = "help"
+        cmd_args = []
     else:
         # Pass any remaining command line arguments to the server.
         cmd_args = args
 
+    if cmd == "server":
+        basedir = os.path.dirname(os.path.abspath(sys.argv[0]))
+        server_location = os.path.join(basedir, "blp-server")
+        if not os.path.isfile(server_location):
+            if options.server_location:
+                server_location = options.server_location
+            else:
+                print("Bloop server could not be located in %s." % server_location)
+                print("Pass in the location with `--server-location` before the `server` command.")
+                sys.exit(1)
+
+        try:
+            check_call(["java", "-jar", server_location])
```

Comment:
Doesn't that mean that people will almost always need to set `--server-location`? Couldn't we get away with only `check_call([server_location])`, defaulting to `blp-server` and assuming it's on the `$PATH`? If it's not on the path, people can still using `bloop --server-location /foo/bar/blp-server server`.

Reply:
I wasn't suggesting checking if this is in path, just pointing out that you don't need the `"java", "-jar", ` part.

It's probably fine to leave it that way.